etna/api/urls/pages.py

```diff
@@ -39,81 +63,11 @@ def listing_view(self, request):
         for restricted_page in restricted_pages:
             queryset = queryset.not_descendant_of(restricted_page, inclusive=True)
 
-        # Check if we have a specific site to look for
```

Comment:
We might need this in the future if we need to serve content for multiple domains. Is there a reason it was removed?